README.md

```diff
@@ -49,13 +72,13 @@ class { 'wsus_client':
 
 ## Usage
 
+### Scheduling Automatic Updates
```

Comment:
The Usage heads should be imperative, and H3 or smaller headings in READMEs should be in sentence case, so this should be:

Schedule automatic updates

The former is an attempt to separate "do a thing" topics from "about a thing" topics, which often use a gerund, and it's the house style for our official docs site docs, too. The sentence case thing is decidedly NOT our usual style, but it is consistent with other modules. We should maybe revisit whether these should be sentence or title case, but for today, it's still sentence.

I know I totally didn't explain either of these issues before; sorry about that!